M2/Macaulay2/packages/Macaulay2Doc/functions/examples-doc.m2

```diff
@@ -46,7 +46,8 @@ Node
     Text
       Certain examples or tests might not be compatible with @TT "capture"@. In such cases,
       use the string @CODE "-* no-capture-flag *-"@ somewhere within @TT "str"@ so that the
-      code is run in a separate Macaulay2 subprocess instead.
+      code is run in a separate Macaulay2 subprocess instead.  In examples, this string
+      should not go on its own line or the documentation will be garbled.
```

Comment:
I don't think this needs to be mentioned in the documentation. It's a caveat that will hopefully be fixed at some point, and there's a good chance we'll forget to remove this line and years later someone (likely us) will wonder why the documentation will be garbled.